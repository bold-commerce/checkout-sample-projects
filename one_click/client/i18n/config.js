import i18n from 'i18next';
import enTranslation from './en/translation.json';
import { initReactI18next } from 'react-i18next';

i18n
  .use(initReactI18next)
  .init({
    resources: {
      en: {
        translation: enTranslation
      }
    },
    lng: 'en',
    fallbackLng: 'en',
    interpolation: {
      escapeValue: false, 
    }
  });

export default i18n;