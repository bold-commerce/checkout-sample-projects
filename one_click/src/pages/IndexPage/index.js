export { default as IndexPage } from "./IndexPage";
export { default as IndexPageGuest } from "./IndexPageGuest";