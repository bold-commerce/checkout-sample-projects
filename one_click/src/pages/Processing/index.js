export { default as ProcessingOrder } from './Processing'