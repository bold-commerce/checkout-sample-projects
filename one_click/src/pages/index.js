export * from './Confirmation';
export * from './IndexPage';
export * from './Processing';
export * from './Inventory';