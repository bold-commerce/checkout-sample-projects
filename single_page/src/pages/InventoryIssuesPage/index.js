export { default as InventoryIssuesPage } from './InventoryIssuesPage';
