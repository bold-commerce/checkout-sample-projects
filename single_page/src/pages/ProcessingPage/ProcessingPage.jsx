import React from 'react';
import { LoadingSpinner } from '@boldcommerce/stacks-ui';
import './ProcessingPage.css';

const ProcessingPage = () => {
  return (
    <div className="Checkout__Processing">
      <LoadingSpinner />
      <h1>Processing order...</h1>
      <p>This may take a few moments. Please remain on the page until the process is complete.</p>
    </div>
  );
};

export default ProcessingPage;
