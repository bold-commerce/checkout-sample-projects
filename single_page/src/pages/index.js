export * from './ConfirmationPage';
export * from './IndexPage';
export * from './ProcessingPage';
