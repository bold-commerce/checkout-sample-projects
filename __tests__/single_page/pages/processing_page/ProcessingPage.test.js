import React from 'react';
import { render } from "@testing-library/react";
import { ProcessingPage } from "../../../../single_page/src/pages/ProcessingPage";
import '../../../../single_page/src/i18n/config';

describe('ProcessingPage', () => {
  test('renders ProcessingPage page', () => {
    const { asFragment } = render( <ProcessingPage /> );

    expect(asFragment()).toMatchSnapshot();
  });
});
