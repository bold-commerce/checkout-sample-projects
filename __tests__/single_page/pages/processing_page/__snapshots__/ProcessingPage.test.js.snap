// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProcessingPage renders ProcessingPage page 1`] = `
<DocumentFragment>
  <div
    class="Checkout__Processing"
  >
    <span
      class="stx-loading-spinner"
    />
    <h1>
      Processing order...
    </h1>
    <p>
      This may take a few moments. Please remain on the the page until the process is complete.
    </p>
  </div>
</DocumentFragment>
`;
